docs/verification.md

```diff
@@ -0,0 +1,146 @@
+    3. Signed by key authorized for the purposes of signing
+    4. Expected properties are present (per proof data model)
+3. Check timeliness and status (per callbacks provided by verifier)
+
+### Verify (VP)
+
+Same as Verify (VC), with these additions/changes:
+
+1. VP proof is optional, but if present it must pass checks.
+2. Make sure the `domain` and `challenge` values are ones the verifier was expecting
+
+
+## Validation
+
+These are categories of validation checks, but do not prescribe specific ways of checking. The verifier may determine how these checks are performed by providing callbacks to the “Verification” layer. This interface is described in "Interfaces between Validation and Verification".
+
+*   Timeliness and Status (callback to verification)
+    *   issuanceDate isn’t in the future, not past expiration date
+    *   Status is “current”, e.g. not revoked
+*   Credential Holder/Subject is expected party
+    *   Authenticate 
+        *   Including DID auth, traditional auth method
+    *   Other checks
+        *   Public key material (TODO: check this doesn't conflict with another category) 
+        *   Subject attributes listed in the credential 
+*   Issuer is expected party
+    *   The value associated with the `issuer` property identifies an issuer that is known to and trusted by the verifier.
+        *   E.g. Check the issuer DID against an allow-list
+*   Proof attributes (TODO: still teasing apart this from DID checks.)
+    *   From: [https://www.w3.org/TR/vc-data-model/#proofs-signatures-0](https://www.w3.org/TR/vc-data-model/#proofs-signatures-0)
+    *   Acceptably recent metadata regarding the public key associated with the signature is available. For example, the metadata might include properties related to expiration, key owner, or key purpose.
+    *   The key is not suspended, revoked, or expired.
+*   DID and DID Document Check (Issuer and Holder)
+    *   A more complete list of considerations should be provided in the context of the DID spec (or related docs). Here are some examples:
+        *   Check to make sure the signing keys are not marked revoked in the latest version of the DID document (or have not been removed).
+            *   If the keys _are_ currently revoked, try and determine whether they were valid at the time the VC was issued, and whether to accept it. (This is very much usecase/threat-model specific).
+            *   Different DID methods and Key types have different ways of marking key expiration/revocation.
+        *   Check to make sure the issuer DID has not been revoked.
+
+
+## Veracity
+
+Perform any other app-specific or domain-specific validation around fitness for purpose. 
+
+Veracity checks do not fall in the scope of vc-http-api, but we call it out as a general bucket for additional checks, which serve one of these purposes
+
+1. Restrict the credentials accepted, to establish credibility, reputation, or “premium brand presence”. Examples include:
```

Comment:
what does "premium brand presence" mean?

Reply:
I reworded it slightly.